pkg/storage/engine/rocksdb/db.cc

```diff
@@ -2306,8 +2306,21 @@ struct DBSstFileWriter {
 };
 
 DBSstFileWriter* DBSstFileWriterNew() {
+  // TODO(dan): Right now, backup is the only user of this code, so that's what
+  // the options are tuned for. If something else starts using it, we'll likely
+  // have to add some configurability.
+
+  rocksdb::BlockBasedTableOptions table_options;
+  // Larger block size (4kb default) means smaller file at the expense of more
+  // scanning during lookups.
+  table_options.block_size = 64 * 1024;
+  // The original LevelDB compatible format.
+  table_options.format_version = 0;
```

Comment:
Link to rocks docs or https://github.com/facebook/rocksdb/blob/master/include/rocksdb/table.h#L198 ?
also, from what I read there, it can be silently converted from 0 to 1... maybe we should explicitly set the checksum too?